client/my-sites/plans/main.jsx

```diff
@@ -100,43 +95,6 @@ var Plans = React.createClass( {
 		}
 	},
 
-	renderTrialCopy: function() {
-		var message,
-			businessPlan,
-			premiumPlan;
-
-		if ( ! this.props.sitePlans.hasLoadedFromServer || getABTestVariation( 'freeTrials' ) !== 'offered' ) {
-			return null;
-		}
-
-		businessPlan = find( this.props.sitePlans.data, isBusiness );
-		premiumPlan = find( this.props.sitePlans.data, isPremium );
-
-		if ( businessPlan.canStartTrial && premiumPlan.canStartTrial ) {
-			message = this.translate( 'Try WordPress.com Premium or Business free for 14 days, no credit card required' );
-		}
-
-		if ( businessPlan.canStartTrial && ! premiumPlan.canStartTrial ) {
-			message = this.translate( 'Try WordPress.com Business free for 14 days, no credit card required' );
-		}
-
-		if ( ! businessPlan.canStartTrial && premiumPlan.canStartTrial ) {
-			message = this.translate( 'Try WordPress.com Premium free for 14 days, no credit card required' );
-		}
-
-		if ( ! businessPlan.canStartTrial && ! premiumPlan.canStartTrial ) {
-			return null;
-		}
-
-		return (
-			<div className="plans__trial-copy">
-				<span className="plans__trial-copy-text">
```

Comment:
We should remove the `plans__trial-copy` and `plans__trial-copy-text` classes from the stylesheet.